generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/_password-reset-finish.component.ts

```diff
@@ -18,16 +18,28 @@ export class PasswordResetFinishComponent implements OnInit {
     resetAccount: any;
     success: string;
     modalRef: NgbModalRef;
+    key: string;
 
-    constructor(private passwordResetFinish: PasswordResetFinish,
+    constructor(
+        <%_ if (enableTranslation) { _%>
+        private jhiLanguageService: JhiLanguageService,
```

Comment:
since this would be on every component we could do it the same way we did earlier for states using copyJs method and by updating the strip regex